src/js/stores/MarathonStore.js

```diff
@@ -217,10 +236,17 @@ var MarathonStore = Store.createStore({
       case ActionTypes.REQUEST_MARATHON_GROUPS_SUCCESS:
         MarathonStore.processMarathonGroups(action.data);
         break;
+      case ActionTypes.REQUEST_MARATHON_DEPLOYMENTS_SUCCESS:
+        MarathonStore.processMarathonDeployments(action.data);
+        break;
       case ActionTypes.REQUEST_MARATHON_GROUPS_ERROR:
         MarathonStore.processMarathonGroupsError();
         break;
+      case ActionTypes.REQUEST_MARATHON_DEPLOYMENTS_ERROR:
+        MarathonStore.processMarathonDeploymentsError();
+        break;
       case ActionTypes.REQUEST_MARATHON_GROUPS_ONGOING:
+      case ActionTypes.REQUEST_MARATHON_DEPLOYMENTS_ONGOING:
```

Comment:
The pattern I'm referring to is putting noop functions everywhere. But in this case, the noop is called `processOngoingRequests` so it could, in theory, process the ongoing requests for both groups and deployments.

Regardless, I think we should not be creating these noops unless there is a good reason to do so.